packages/itwinui-css/src/workflow-diagram/workflow-diagram.scss

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../stepper/index';
+
+@mixin iui-workflow-diagram {
+  @include iui-stepper;
+
+  > ol {
```

Comment:
What is the purpose of adding a wrapper div around the `<ol>`? Can't we apply all styles on the `<ol>` directly?

Reply:
I am able to remove it.  I added a `@mixin iui-stepper-list` in `stepper.scss` so I could reuse the necessary parts of `@mixin iui-stepper` within workflow diagram.